### PR TITLE
feat: update skeleton loader colors

### PR DESCRIPTION
**Pull request by mkrl**
[FX-3824](https://toptal-core.atlassian.net/browse/FX-3824)

### Description

Updated the color gradient for Skeleton loaders so it is usable on all backgrounds (well, apart from one specific grey one)

### How to test

- Take a look at the visual tests
- Take a look at the deployed Storybook, go to `SkeletonLoader` and scroll down to the [latest story](https://picasso.toptal.net/fx-3824-skeleton-loaders-loaders-on-dark-backgrounds/?path=/story/components-skeletonloader--skeletonloader#different-background-color) added for dark backgrounds
- See if the new shared colors for loaders make sense (the location in the code and naming) 

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-3824]: https://toptal-core.atlassian.net/browse/FX-3824?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/picasso-provider/src/Picasso/config/palette.ts [hunk 1]
```diff
@@ -68,6 +70,11 @@ export const colors = {
   },
 }
 
+export const loaderColors = {
+  foreground: alpha(colors.grey.main2, 0.16),
```
Comment (TomasSlama) [inline packages/picasso-provider/src/Picasso/config/palette.ts:74]: do we really need these colors in the provider?, Withi this change we should bump peer dependency of the provider in the Picasso resulting in major release of Picasso. I think the provider should be as stable as possible with as few changes as possible. 🤔 Maybe having these only in Picasso utils might be enough? 🤔

**Review by TomasSlama — COMMENTED** [review]
tested and temploy and it works as expected, left one comment

File: packages/picasso-provider/src/Picasso/config/palette.ts (hunk 1 above)
Comment (mkrl) [reply packages/picasso-provider/src/Picasso/config/palette.ts:74]: Good point, I was unsure if it's place good enough for storing it. It's shared between all the skeleton loaders and not used anywhere else, I'll move it to be contained in utils only.

Comment (mkrl) [reply packages/picasso-provider/src/Picasso/config/palette.ts]: Updated

File: packages/picasso/src/utils/loader-palette.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { alpha } from '@material-ui/core'
```
Comment (TomasSlama) [inline packages/picasso/src/utils/loader-palette.ts:1]:
```suggestion
import { alpha } from '@toptal/picasso-shared'
```

**Review by TomasSlama — APPROVED** [review]
one small comment, but otherwise it looks great!

**Pull request merged**